File: S13_20-10-2022/Inheritance.js
class A {
  name = "Kabir Sagar";
  city = "Delhi";

  printPersonalDetails() {
    console.log(this.name, this.city); //100
  }
}

class B extends A {
  email = "KS@gmail.com";
  phone = 99999999;

  details() {
    this.name = "Darsh";

    this.printPersonalDetails();
    console.log(this.email, this.phone);
  }
}

var b1 = new B();
b1.details();

File: S22_10-11-2022/Inheritance.java
class A {
    int x = 10;
    int y = 20;

    public int sum() {
        return x + y;
    }

    public static void main(String[] args) {
        A a1 = new A();
        System.out.println(a1.sum());
    }

}

class B extends A {
    String name = "Darsh";
    String city = "Delhi";

    public void displayData() {
        System.out.println(name);
        System.out.println(city);
    }

    public static void main(String[] args) {
        B b1 = new B();
        System.out.println(b1.sum());
        b1.displayData();
    }

}
